src/main/java/tatracker/model/module/Module.java

```diff
@@ -55,13 +55,27 @@ public String getName() {
         return name;
     }
 
+    /**
+     * Returns module at index n.
+     */
+    public Group get(int n) {
```

Comment:
getGroup maybe? Or does this clash with smth else?

Reply:
Yes, we have a get group. Plus i thought this would make sense intuitively since it gives the group at index n, like array lists.